### DISCUSSION TITLE
How to config json format for keeping multiple lines for array type fields?

### DISCUSSION THREAD

**Discussion by mortal-dog** [biomejs/biome#4249] [Q&A] [answered]
I have source json file
```
{
  "files": [
    "src/", 
    "public/"
  ]
}
```
Biome always format them into a single line. 
```
{
  "files": ["src/", "public/"]
}
```
Is there a way to config biome, so that I can keep multiple lines for array type fields ?

**Comment by arendjr** [comment]
Sorry, we follow the [Prettier philosophy](https://prettier.io/docs/en/option-philosophy) in this regard, so there's no configuration for this.

**Comment by ematipico** [comment, marked as answer]
You can use line width to 1. That does the trick

**Reply by mortal-dog** [reply]
That works !
It looks like a bug, but anyway, it works for me. Big thanks !
